bin/spark-connect

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+if [ -z "${SPARK_HOME}" ]; then
+  source "$(dirname "$0")"/find-spark-home
+fi
+
+# Build the jars needed for spark submit
+#build/sbt package
+
+# Build the jars needed for spark connect
+build/sbt "connect/assembly"
+
+CONNECT_JAR=`ls "${SPARK_HOME}"/connector/connect/server/target/scala*/spark-connect-assembly*.jar | paste -sd ',' -`
+
+exec "${SPARK_HOME}"/bin/spark-submit --class org.apache.spark.sql.connect.SimpleSparkConnectService "$CONNECT_JAR"
```

Comment:
We have two scripts for starting and stopping a connect server. Should we use those instead?

Reply:
Where is the other script?